### PR TITLE
Improved ODQA model with squad logits.

### PR DESCRIPTION
**Pull request by my-master**
Improved ODQA model, f1 raised from 14 to 26. For English and Russian languages.

Features:

- odqa preprocessors classes
- logit ranker class
- updated main and test configs
- updated documentation
- **new** enwiki dump and squad models (requires downloading from server)

### CODE REVIEW

File: deeppavlov/configs/odqa/ru_odqa_infer_wiki.json [hunk 1]
```diff
@@ -29,18 +25,30 @@
...
+        "flatten_result": true
+      },
+      {
+        "name": "string_multiplier",
+        "in": ["question_raw", "chunks"],
+        "out":["questions"]
+      },
+      {
+        "name": "logit_ranker",
+        "squad_model_config": "configs/squad/squad_ru.json",
```
Comment (yoptar) [inline deeppavlov/configs/odqa/ru_odqa_infer_wiki.json]: `"squad_model": {"config_path": "../deeppavlov/configs/squad/squad_ru.json"}`?

File: deeppavlov/models/ranking/logit_ranker.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+     Raises:
+         FileNotFoundError if a relative path to :attr:`squad_model_config` doesn't have "configs" module as root
+
+    """
+
+    def __init__(self, squad_model_config: str, **kwargs):
+        if Path(squad_model_config).is_absolute():
+            abs_config_path = squad_model_config
+        else:
+            abs_config_path = Path(__file__, "..", "..", "..", squad_model_config).resolve()
```
Comment (yoptar) [inline deeppavlov/models/ranking/logit_ranker.py]: All relative paths in configs should be relative to deeppavlov_root. It can be resolved using `deeppavlov.core.commands.utils.expand_path(path)`

File: deeppavlov/vocabs/wiki_sqlite.py [hunk 3]
```diff
@@ -28,21 +28,25 @@ class WikiSQLiteVocab(SQLiteDataIterator):
...
-    """
+    Attributes:
+        join_docs: whether to join extracted docs with ' ' or not
 
-    def __init__(self, data_url: str, data_dir: str = '', **kwargs):
+    """
 
-        super().__init__(data_dir=data_dir, data_url=data_url)
+    def __init__(self, data_url: str, data_dir: str = '', join_docs: bool=True, **kwargs):
+        super().__init__(data_dir=data_dir, data_url=data_url, shuffle=kwargs.get('shuffle', False))
```
Comment (yoptar) [inline deeppavlov/vocabs/wiki_sqlite.py]: Why not add `shuffle=False` to `__init__()` parameters then?

File: deeppavlov/configs/odqa/ru_odqa_infer_wiki.json (hunk 1 above)
Comment (my-master) [reply deeppavlov/configs/odqa/ru_odqa_infer_wiki.json]: ok

File: deeppavlov/models/ranking/logit_ranker.py (hunk 2 above)
Comment (my-master) [reply deeppavlov/models/ranking/logit_ranker.py]: ok

File: deeppavlov/vocabs/wiki_sqlite.py (hunk 3 above)
Comment (my-master) [reply deeppavlov/vocabs/wiki_sqlite.py]: ok

File: deeppavlov/configs/odqa/en_odqa_infer_wiki.json [hunk 4]
```diff
@@ -55,9 +63,17 @@
       "server_utils": "ODQA"
     },
     "download": [
+      "http://files.deeppavlov.ai/deeppavlov_data/squad_model_1.3_cpu_compatible.tar.gz",
```
Comment (yoptar) [inline deeppavlov/configs/odqa/en_odqa_infer_wiki.json]: When `"config_path"` parameter is used, download links are found recursively, so this link would be found in `squad.json` and is not needed here.
